packages/ethereum-contracts/dev-scripts/deploy-test-framework.js

```diff
@@ -141,9 +170,20 @@ const deployTestFramework = async () => {
                 SuperfluidPeripheryDeployerLibrary:
                     SuperfluidPeripheryDeployerLibrary.address,
                 SuperTokenDeployerLibrary: SuperTokenDeployerLibrary.address,
+                SuperfluidNFTLogicDeployerLibrary:
+                    SuperfluidNFTLogicDeployerLibrary.address,
+                ProxyDeployerLibrary: ProxyDeployerLibrary.address,
+                CFAv1ForwarderDeployerLibrary:
+                    CFAv1ForwarderDeployerLibrary.address,
+                SuperfluidLoaderDeployerLibrary:
+                    SuperfluidLoaderDeployerLibrary.address,
             },
         }
     );
+    const numSteps = await frameworkDeployer.getNumSteps();
+    for (let i = 0; i < numSteps; i++) {
+        await frameworkDeployer.executeStep(i);
```

Comment:
I couldn't find where this executeStep is defined :D ??

Reply:
it's in `SuperfluidFrameworkDeploymentSteps`